### PR TITLE
chore(docs): add sprint 1 documentation

### PR DESCRIPTION
**Pull request by DavidLiang01**
-software architecture document
-prototypes
-risk assessment

### CODE REVIEW

**Review by Mackbellemore — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -42,57 +49,57 @@ volumes:
...
 
-2.  Rebuild the project using `make run-server` or `docker-compose up --build server`
-   
+2. Rebuild the project using `make run-server` or `docker-compose up --build server`
+
 3. Start by opening the the server folder in its on VS Code window.
-   
+
 4. In VS code on the debug tab click on "create launch.json" or click on the gear icon. Add the following to `"configurations:"` :
+
```
Comment (Mackbellemore) [inline README.md:60]: can you remove this section and just point to the json file i pushed ples

Comment (Mackbellemore) [reply README.md:60]: das my bad i forgot to do that

Comment (Mackbellemore) [reply README.md:60]: actually its chill ill push it i need to make some adjustments to it

**Review by Mackbellemore — DISMISSED** [review]

**Comment by gordonpn** [comment]
Link:
* [ ] Testing plan
* [ ] Risk management
* [ ] Figma view only link (?)

File: README.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Software Engineering Team Design Project
 
+## Protoype
+[Click here to view Figma prototype](https://www.figma.com/file/1NeYMymzl5OYosLV0D24rM/PROTOTYPE)
+
+## Reports
+
```
Comment (nasjuice) [inline README.md:7]: do we need a link for this ?

Comment (Mackbellemore) [reply README.md:7]: a link for what?

Comment (nasjuice) [reply README.md:7]: reports shouldnt we combine all of the docs into one report and have a link for it

**Review by Mackbellemore — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
 # Software Engineering Team Design Project
+## Reports
+
+[Link to Sprint 1 Report](https://github.com/Mackbellemore/soen-390-team07/blob/develop/docs/sprint1_report.md)
```
Comment (gordonpn) [inline README.md:4]:
```suggestion
[Link to Sprint 1 Report](./docs/sprint1_report.md)
```

We can use relative link to make the link valid on any branch (blobs)

**Review by nasjuice — DISMISSED** [review]

File: README.md (hunk 3 above)
Comment (Mackbellemore) [reply README.md:4]: ah ya my b

**Review by Mackbellemore — APPROVED** [review]

**Review by gordonpn — APPROVED** [review]

**Pull request merged**